src/Pie.js

```diff
@@ -77,15 +76,15 @@ const RoundDividers = ({ paintedSections, dividerSize, width, radius, background
 };
 
 const getArcAngle = (percentage) => percentage / 100 * 360;
-const shouldShowDivider = (percentage, dividerSize) => percentage <= 100 && !Number.isNaN(dividerSize);
+const shouldShowDivider = (sections) => sections.length > 1;
```

Comment:
The `!Number.isNaN(dividerSize)` check was originally here for the above comment I made so it might be needed back

Reply:
If we are handling it here then we don't need any code changes inside `RoundDivider` function.